include/ua_plugin_nodestore.h

```diff
@@ -487,6 +487,10 @@ UA_Node_setAttributes(UA_Node *node, const void *attributes,
 UA_StatusCode UA_EXPORT
 UA_Node_copy(const UA_Node *src, UA_Node *dst);
 
+/* Allocate new node and copy the values from src */
+UA_StatusCode UA_EXPORT
+UA_Node_copy_alloc(const UA_Node *src, UA_Node **dst);
```

Comment:
UA_Node_copy_alloc could return a pointer to the new node instead of going with double-pointers.